content/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.md

```diff
@@ -93,27 +93,25 @@ The helpers are split into three folders:
-| `3-buildCPIs`                         | Builds your CorDapp and wraps it in a signed CPI.                                                      |
-| `4-deployCPIs`                        | Deploys the CPI to your local Corda cluster.                                                           |
-| `5-createAndRegisterVNodes`           | Sets up the Virtual Nodes specified in `config/dev-net.json` on you local corda with the uploaded CPI. |
-| `quickDeployCordapp`           | An alias for `5-createAndRegisterVNodes`. Run this to perform your first deployment of your CorDapp to a newly started cluster. |
+| <div style="width:220px">Helper</div> | Description                                                                                                                                                                             |
+|---------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `1-createGroupPolicy`                 | Creates the Group policy which is required to set up the Application Network.                                                                                                           |
+| `2-createKeyStore`                    | Creates the signing keys for publishing the CPIs.                                                                                                                                       |
+| `3-buildCPIs`                         | Builds your CorDapp and wraps it in a signed CPI.                                                                                                                                       |
+| `4-deployCPIs`                        | Deploys the CPI to your local Corda cluster. If the CPI has already been deployed to the nodes on the cluster it will perform a forced upload and replace the old CPI with the new one. |
+| `5-vNodeSetup`                        | Sets up the Virtual Nodes specified in `config/static-network-config.json` on you local Corda cluster with the uploaded CPI.                                                            |
 
 Each of these tasks has a dependency on the previous. So, if you run 3, it also runs 1 and 2.
 
 {{< note >}}
-You only need to run `quickDeployCordapp` the first time you upload your CPI to the corda cluster. On subsequent builds, you can just run `4-deployCPIs`. This reduces your turnaround time for deploying your code from 1-2 mins to around 20-30 seconds.
-{{< /note >}}
+You only need to run `5-vNodeSetup` the first time you upload your CPI to the corda cluster. On subsequent builds, you can just run `4-deployCPIs`. (Note, `5-vNodeSetup` would have no effect if the virtual nodes have already been set up, it will not try to recreate them.)
```

Comment:
```suggestion
You only need to run `5-vNodeSetup` the first time you upload your CPI to the corda cluster. On subsequent builds, you can just run `4-deployCPIs`. (Note, `5-vNodeSetup` has no effect if the virtual nodes have already been set up. It will not try to recreate them.)
```